.github/workflows/release.yml

```diff
@@ -32,28 +32,24 @@ jobs:
           echo "::set-output name=version::$(git describe --tags --always --dirty | cut -c2-)"
 
   builder:
+    name: builder-${{matrix.os}}-${{matrix.arch}}
     needs: [args]
+    strategy:
+      matrix:
+        os:
+          - linux
+          - windows
+          - darwin
+        arch:
+          - amd64
+          - arm64
     permissions:
       actions: read # For the detection of GitHub Actions environment.
       id-token: write # For signing.
       contents: write # For asset uploads.
     uses: slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@v1.4.0
     with:
       go-version: 1.18
-      config-file: .github/config-release.yml
+      config-file: .slsa-goreleaser-${{matrix.os}}-${{matrix.arch}}.yml
```

Comment:
Could you move the config files into a new .goreleaser folder? Sorry, I did not catch this earlier. Thanks!

Reply:
These configuration files are not for GoReleaser, so I prefer `.slsa-goreleaser` than `.goreleaser`.
How about `.slsa-goreleaser/${{matrix.os}}-${{matrix.arch}}.yml`?
What do you think?

Reply:
SGTM

Reply:
Moved. 477c130cfa318e7c5c01d9f75cb8810d2a2d6b41

Reply:
It works well.
https://github.com/suzuki-shunsuke/slsa-verifier/releases/tag/v2.0.2-rc.4
https://github.com/suzuki-shunsuke/slsa-verifier/actions/runs/3859552606